src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -95,16 +94,16 @@ object DockerPlugin extends AutoPlugin {
 
   override lazy val projectSettings: Seq[Setting[_]] = Seq(
     dockerAlias := DockerAlias(
-      (dockerRepository in Docker).value,
-      (dockerUsername in Docker).value,
-      (packageName in Docker).value,
-      Option((version in Docker).value)
+      (Docker / dockerRepository).value,
+      (Docker / dockerUsername).value,
+      (Docker / packageName).value,
+      Option((version).value)
     ),
-    dockerLayerGrouping := { _: String =>
+    Docker / dockerLayerGrouping := { _: String =>
```

Comment:
Sorry for the inconsistency here. The tasks / settings already prefixed with the format, e.g. `dockerLayerGrouping` should not be scoped. This will break clients as settings are not applied anymore.


```suggestion
    dockerLayerGrouping := { _: String =>
```